apps/docs/src/.vitepress/components/DesignVariable.vue

```diff
@@ -97,10 +102,17 @@ const emit = defineEmits<{
     &--no-value {
       padding: var(--onyx-spacing-3xs) var(--onyx-spacing-2xs);
     }
+    &--no-padding {
+      padding: 0;
+    }
   }
 
   &__value {
+    color: var(--onyx-color-base-neutral-500);
     font-weight: 600;
+    font-size: 0.8125rem;
```

Comment:
(optional) we also have a `onyx-text--small` class for this so we don't need to remember this weird rem value all the time :D